app/controllers/users/sessions_controller.rb

```diff
@@ -123,6 +123,15 @@ def auth_params
       params.require(:user).permit(:email, :password, :request_id)
     end
 
+    # Overrides a method from Devise::SessionController to handle malformed params
+    def sign_in_params
```

Comment:
Do we need to do the `super` on line 27? I don't quite get what Devise is doing with the parameters here.